boranga/frontend/boranga/src/components/internal/species_communities/dashboard.vue

```diff
@@ -32,9 +51,11 @@ export default {
     data() {
         let vm = this;
         return {
-            user_preference:'flora',
-            filterGroupType: 'fauna', // TODO : set it to default user preference but for now is hardcoded value
+            user_preference:'flora',    // TODO : set it to default user preference but for now is hardcoded value
+            filterGroupType: 'flora',  // TODO : need to set to default user preferance as cannot call click event of Tab onload
```

Comment:
Can we ask Denis to create a task for this or add these TODOs to an existing task so we:
1. Don't forget .
2. Have time allocated to do it.

Reply:
Yes we can refer this to Dennis and create a seperate task if not yet.